src/components/SearchAndFilter/SearchAndFilter.stories.mdx

```diff
@@ -4,7 +4,16 @@ import SearchAndFilter from "./SearchAndFilter";
 
 <Meta title="SearchAndFilter" component={SearchAndFilter} />
 
-export const Template = (args) => <SearchAndFilter {...args} />;
+export const Template = (args) => (
+  <SearchAndFilter
+    {...args}
+    returnSearchData={(searchData) => {
```

Comment:
If the `returnSearchData` prop is required, it should be marked required in the component.
If it's not required, component should render without it.

Reply:
Good point, I've now cleaned that up.